js/tji-datasets-explore.js

```diff
@@ -544,24 +557,60 @@ TJIChartView.prototype.create_charts = function() {
         chart_constructor = TJIGroupByBarChart;
         break;
     }
+// *  $container: jQuery object to contain chart
+// *  group_by: column to group by
+// *  missing_data_label: stand-in label for records missing
+// *                       the group_by column
+// *  data: list of objects representing records
     that.components.charts.push(
       new chart_constructor({
         $container: $container, 
         group_by: config.group_by, 
         sort_by: config.sort_by,
         missing_data_label: that.missing_data_label,
-        data: that.state.chart_data,
+        data: dataset.chart_data,
       })
     );    
   });
 }
 
-// $container: jQuery object to contain chart
-// *   groupBy: column to group by
-// *   missing_data_label: stand-in label for records missing
-// *                       the groupBy column.
-// *   data: list of objects representing records
+TJIChartView.prototype.create_chartview_DOM = function() {
+  var that = this;
+
+  this.ui.$loader = jQuery('<div />', {
+    class: 'tji-chartview__loader'
+  }).appendTo(this.ui.$chartview_charts);
 
+  this.ui.$form = jQuery('<form />', {
+    class: 'tji-chart-filters',
+  }).appendTo(this.ui.$chartview_filters);
+
+  this.ui.$charts = jQuery(this.templates.chartview_charts_template)
+    .addClass('tji-chartview__charts')
+    .appendTo(this.ui.$chartview_charts);
+
+  this.ui.$select_dataset = jQuery('<select />', {
```

Comment:
Yep, I haven't addressed styling at all in this branch. You'll see in the description of this PR it's an open to do, especially since Jason was working on this independently. I was only worried about js functionality. I'll make another branch with styling.